m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -983,38 +1004,27 @@ END [PROCEDURE]
 
```

Comment:
Hi @tkainz ,The code looks good.
I have one question though for the following code for line 914-918, do you need to call CHECKFORINVALIDCHARS again on NEWTRANSLDESCRIPTION, as you've already called each time that variable came up in the rest of the code?

 CHARCHECK=NEWTRANSLDESCRIPTION
 CALL CHECKFORINVALIDCHARS
 NEWTRANSLDESCRIPTION=CHARCHECKRETURN

 PRINT INDENT(BASEINDENT+3)+Q+"toAccountName"+Q+": "+Q+NEWTRANSLDESCRIPTION+Q+","

Reply:
@jordan-keenan Thanks - good point.  Corrected.

Reply:
Thanks @tkainz , Changes from end of day 10/14 also look good!